src/main/java/uk/gov/pay/adminusers/resources/ServiceRequestValidator.java

```diff
@@ -89,6 +93,16 @@ private boolean nonNumericGatewayAccountIds(JsonNode gatewayAccountNode) {
         return Optional.empty();
     }
 
+    public Optional<Errors> validateUpdateMerchantDetailsRequest(JsonNode payload) {
+        Optional<List<String>> missingMandatoryFields = requestValidations.checkIfExists(payload,
```

Comment:
I needed to check what does the method _checkIfExists_ because it does not only check the presence but the value is not blank. So maybe to consider renaming the method in future PR or creating a _RequestValidations_ test that reflects the intentions. 

Not for this PR but something we can consider to do later.

Reply:
Thanks, addressed in [7db158c]